include/RAJA/policy/tensor/arch/avx/avx_int32.hpp

```diff
@@ -229,7 +229,7 @@ namespace expt
        */
       RAJA_INLINE
       self_type const &store_packed_n(element_type *ptr, camp::idx_t N) const{
-        _mm256_maskstore_ps(reinterpret_cast<float*>(ptr), createMask(N), m_value);
+        _mm256_maskstore_ps(reinterpret_cast<float*>(ptr), createMask(N), reinterpret_cast<__m256>(m_value));
```

Comment:
ay carumba!

Reply:
agreed!

Reply:
Good catch, intel19 also complained about this

Reply:
Perhaps our CI isn't compiling with SIMD instructions turned on?  we should probably throw the "-march=native" for that

Reply:
I'll work with Mike to see if this can be done with our current azure CI and also gitlab CI.